### PR TITLE
Revert to older, v.2.9, build_config.

### PR DESCRIPTION
**Pull request by LaakkonenJussi**
The build_config in v.2.10 broke AP mode on some devices, the network was not visible. It seems that the old build_config makes this work. This simply reverts the changes made and further work is needed to figure out what exactly breaks the AP mode.

### CODE REVIEW

**Comment by pvuorela** [comment]
To be honest I like the old config with the actual change to defaults better.

However on the last update some keys were reported changed, commit 10ff30c134fd5d3

Hence could remove from here CONFIG_IEEE80211N, CONFIG_PEERKEY and CONFIG_CTRL_IFACE_DBUS which should no longer exist.

**Review by mlehtima — APPROVED** [review]
Seems to work now

File: rpm/build-config [hunk 1]
```diff
@@ -1,643 +1,43 @@
...
-# yet secure randomness source than /dev/random on Linux 3.17 and newer.
-# Requires glibc 2.25 to build, falls back to /dev/random if unavailable.
-CONFIG_GETRANDOM=y
-
-# IEEE 802.11ac (Very High Throughput) support (mainly for AP mode)
+#Added 2019-05-21
+CONFIG_TLS=openssl
 CONFIG_IEEE80211AC=y
-
 CONFIG_IEEE80211W=y
```
Comment (pvuorela) [inline rpm/build-config]: This seems now gone from the code and could be removed as well.

**Review by pvuorela — APPROVED** [review]
Did a quick git grep for the remaining keys and found one more which doesn't exist and could be removed.

Otherwise think this is good. I'll just approve already.

File: rpm/build-config (hunk 1 above)
Comment (LaakkonenJussi) [reply rpm/build-config]: Done.

**Pull request merged**